src/main/java/pl/javastart/exercise/streams/FilteringMachine.java

```diff
@@ -0,0 +1,35 @@
+package pl.javastart.exercise.streams;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FilteringMachine {
+
+    public List<Integer> filterOutNotEvenNumbers(List<Integer> numberList) {
+        return numberList.stream()
+                .filter(number -> number % 2 == 0)
+                .collect(Collectors.toList());
+    }
+
+    public List<Integer> filterOutLowerNumbersThan20(List<Integer> numberList) {
+        return numberList.stream()
+                .filter(number -> number > 20)
```

Comment:
Chcesz przefiltrować liczby MNIEJSZE od 20. 20 nie jest liczbą mniejszą od 20. Tak więc tutaj powinien być znak >=

Reply:
Poprawiłem będzie to widać w następnym commicie.